waspc/data/Generator/templates/server/src/middleware/operations.ts

```diff
@@ -4,13 +4,14 @@ import {
     serialize as superjsonSerialize,
 } from 'superjson'
 import { handleRejection } from 'wasp/server/utils'
+import { enrichAuthUser } from 'wasp/auth/user'
 
 export function createOperation (handlerFn) {
     return handleRejection(async (req, res) => {
         const args = (req.body && superjsonDeserialize(req.body)) || {}
         const context = {
             {=# isAuthEnabled =}
-            user: req.user
+            user: req.user ? enrichAuthUser(req.user) : null,
```

Comment:
This piece of code also change the type from `undefined` to `null` when the user is not authenticated.